src/Icon/iconsList.ts

```diff
@@ -0,0 +1,415 @@
+  'policy-doc': PolicyDoc,
+  'policy-docs': PolicyDocs,
+  'pound-bold': PoundBold,
+  'pound-filled-circle': PoundFilledCircle,
+  'pound-medium': PoundMedium,
+  'pound-outline': PoundOutline,
+  'pound-regular': PoundRegular,
+  'question-help': QuestionHelp,
+  'refer-a-friend': ReferAFriend,
+  'reg-plate': RegPlate,
+  'renewal-line': RenewalLine,
+  'repeat-charge': RepeatCharge,
+  'schedule-quote': ScheduleQuote,
+  'send-message': SendMessage,
+  'shield-check': ShieldCheck,
+  'thumbs-up': ThumbsUp,
+  'tow-truck': TowTruck,
+  'trace-and-access': TraceAndAccess,
+  'up-down': UpDown,
+  'upload-documents': UploadDocuments,
+  'upload-email': UploadEmail,
+  'wheel-flat': WheelFlat,
+  'windscreen-repair': WindscreenRepair,
+  'windscreen-replacement': WindscreenReplacement,
+  aa: Aa,
+  addons: AddOns,
+  alert: Alert,
+  amex: Amex,
+  arrow: Arrow,
+  at: At,
+  axa: Axa,
+  bank: Bank,
+  basket: Basket,
+  bicycle: Bicycle,
+  boost: Boost,
+  breakdown: Breakdown,
+  briefcase: Briefcase,
+  burger: Burger,
+  calendar: Calendar,
+  camera: Camera,
+  cancel: Cancel,
+  car: Car,
+  card: Card,
+  caret: Caret,
+  charge: Charge,
+  chat: Chat,
+  checkout: Checkout,
+  city: City,
+  claim: Claim,
+  clock: Clock,
+  confused: Confused,
+  copy: Copy,
+  crop: Crop,
+  cross: Cross,
+  download: Download,
+  earth: Earth,
+  edit: Edit,
+  engine: Engine,
+  europe: Europe,
+  excluded: Excluded,
+  expand: Expand,
+  facebook: Facebook,
+  fence: Fence,
+  file: File,
+  filter: Filter,
+  fire: Fire,
+  flag: Flag,
+  folder: Folder,
+  fuel: Fuel,
+  globe: Globe,
+  gpay: Gpay,
+  house: House,
+  id: Id,
+  included: Included,
+  info: Info,
+  injury: Injury,
+  instagram: Instagram,
+  intercom: Intercom,
+  intercom2: Intercom2,
+  iphone: Iphone,
+  jewellery: Jewellery,
+  key: Key,
+  laptop: Laptop,
+  leaks: Leaks,
+  lightbulb: Lightbulb,
+  link: Link,
+  linkedin: Linkedin,
+  location: Location,
+  marshmallow: Marshmallow,
+  mastercard: Mastercard,
+  maximise: Maximise,
+  medical: Medical,
+  minimise: Minimise,
+  minus: Minus,
+  mulsanne: Mulsanne,
+  notes: Notes,
+  notification: Notification,
+  onfido: Onfido,
+  other: Other,
+  padlock: Padlock,
+  people: People,
+  person: Person,
+  photo: Photo,
+  plus: Plus,
+  policy: Policy,
+  premfina: Premfina,
+  profile: Profile,
+  prohibited: Prohibited,
+  property: Property,
+  rac: Rac,
+  ravelin: Ravelin,
+  refund: Refund,
+  renewal: Renewal,
+  return: Return,
+  search: Search,
+  security: Security,
+  settings: Settings,
+  shield: Shield,
+  shopping: Shopping,
+  sofa: Sofa,
+  stack: Stack,
+  storm: Storm,
+  stripe: Stripe,
+  subsidence: Subsidence,
+  theft: Theft,
+  tick: Tick,
+  training: Training,
+  trash: Trash,
+  twitter: Twitter,
+  upload: Upload,
+  vandalism: Vandalism,
+  village: Village,
+  visa: Visa,
+  warning: Warning,
+  watch: Watch,
+  wave: Wave,
+  wellbeing: Wellbeing,
+  windscreen: Windscreen,
```

Comment:
I would probably say for consistency we should have these in quotes even though it's not strictly needed.